developer-docs-site/docs/guides/handle-aptos-errors.md

```diff
@@ -0,0 +1,164 @@
+|SIGNERS_CONTAIN_DUPLICATES|There are duplicates among signers, including the sender and all the secondary signers.|
+|SEQUENCE_NONCE_INVALID|The sequence nonce in the transaction is invalid (too new, too old, or already used).|
+|CHAIN_ACCOUNT_INFO_DOES_NOT_EXIST|There was an error when accessing chain-specific account information.|
+|MODULE_ADDRESS_DOES_NOT_MATCH_SENDER|he self address of a module the transaction is publishing is not the sender address.|
+|ZERO_SIZED_STRUCT|Reported when a struct has zero fields.|
+|DUPLICATE_MODULE_NAME|The sender is trying to publish two modules with the same name in one transaction.|
+|BACKWARD_INCOMPATIBLE_MODULE_UPDATE|The sender is trying to publish a module that breaks the compatibility checks.|
+|CYCLIC_MODULE_DEPENDENCY|The updated module introduces a cyclic dependency (i.e., A uses B and B also uses A).|
+|INVALID_FRIEND_DECL_WITH_SELF|Cannot mark the module itself as a friend.|
+|INVALID_FRIEND_DECL_WITH_MODULES_OUTSIDE_ACCOUNT_ADDRESS|Cannot declare modules outside of account address as friends.|
+|INVALID_FRIEND_DECL_WITH_MODULES_IN_DEPENDENCIES|Cannot declare modules that this module depends on as friends.|
+|CYCLIC_MODULE_FRIENDSHIP|The updated module introduces a cyclic friendship (i.e., A friends B and B also friends A).|
+|INVALID_PHANTOM_TYPE_PARAM_POSITION|A phantom type parameter was used in a non-phantom position.|
+|LOOP_MAX_DEPTH_REACHED|Loops are too deeply nested.|
+|TYPE_RESOLUTION_FAILURE|Failed to resolve type due to linking being broken after verification.|
+|RESOURCE_DOES_NOT_EXIST|We tried to access a resource that does not exist under the account.|
+|RESOURCE_ALREADY_EXISTS|We tried to create a resource under an account where that resource already exists.|
+|UNKNOWN_STATUS|A reserved status to represent an unknown vm status. This is std::u64::MAX, but we can't pattern match on that, so put the hardcoded value in.|
+|
+|
+
+## Move Standard Library (stdlib)
+
+|Error |Meaning  |
+--- | :---: |
+|INVALID_ARGUMENT|Caller specified an invalid argument (http: 400).|
+|OUT_OF_RANGE|An input or result of a computation is out of range (http: 400).|
+|INVALID_STATE|The system is not in a state where the operation can be performed (http: 400).|
+|UNAUTHENTICATED|Request not authenticated due to missing, invalid, or expired auth token (http: 401).|
+|PERMISSION_DENIED|The client does not have sufficient permission (http: 403).|
+|NOT_FOUND|A specified resource is not found (http: 404).|
+|ABORTED|Concurrency conflict, such as read-modify-write conflict (http: 409).|
+|ALREADY_EXISTS|The resource that a client tried to create already exists (http: 409).|
+|RESOURCE_EXHAUSTED|Out of gas or other forms of quota (http: 429).|
+|CANCELLED|Request cancelled by the client (http: 499).|
+|INTERNAL|Internal error (http: 500).|
+|NOT_IMPLEMENTED|Feature not implemented (http: 501).|
+|UNAVAILABLE|The service is currently unavailable. Indicates that a retry could solve the issue (http: 503).|
```

Comment:
I'm not quite sure here, they aren't http codes, is this supposed to be a reference to similar behavior?

Reply:
Oh is this supported to be HTTP codes?  maybe the type grouping is wrong

Reply:
> Oh is this supported to be HTTP codes? maybe the type grouping is wrong

These came from:
https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/move-stdlib/sources/error.move#L24

What do you think we should do?

Reply:
Greg, can we get this in for launch?

Reply:
Oh, that's super confusing.  Sorry I missed this comment